R/section.R

```diff
@@ -1071,6 +1071,9 @@ sectionAddCtd <- sectionAddStation
 #' in accordance with the resolution of the latitudes in the `topo` object.
 #' See \dQuote{Examples}.
 #'
+#' @param drawPalette Logical value indicating whether to draw a palette when `ztype="image"`
+#' ignorned otherwise.
```

Comment:
spelling error

Reply:
@clayton33 any chance you could fix the spelling error ('ignored') and do a second PR?  (Otherwise, git-blame will say that I wrote the line, because git tracks by line and not character.)